python/foglamp/plugins/south/coap_listen/coap_listen.py

```diff
@@ -114,8 +114,14 @@ def plugin_reconfigure(handle, new_config):
         new_handle: new handle to be used in the future calls
     Raises:
     """
+    # TODO: handle vs new_config
+    # only use case for difference of old and new config or anything else?
 
-    new_handle = {}
+    _LOGGER.info("Old config for Coap plugin {} \n new config {}".format(handle, new_config))
+    new_handle = plugin_init(new_config)
+
+    # TODO: Right now plugin_shutdown has no body, needed before start - FOGL-Story?
```

Comment:
We only need this if the plugin has some actions to do on shutdown. In the case of Coap we might want to close sockets or the like.